### PR TITLE
Added files for verifying [Policies]: lack of "user_consent_prompt" field is included to the <appID>`s policies

### PR DESCRIPTION
**Pull request by aarotar**

### CODE REVIEW

**Comment by aarotar** [comment]
@maria-mih, please review

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+--
+-- Description: 
+--     PTU for <appID> contains policies with <functional grouping> that lacks "user_consent_prompt" field
+--     1. Used preconditions:	
+--			Unregister default app
+--			Register application
+--			Consent device-> Activate application
+--
+--     2. Performed steps
+--			Perform PTU that contains policies for app with <functional grouping> that lacks "user_consent_prompt" field
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: Shouldn't this be in the in the preconditions as well?

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+--			Unregister default app
+--			Register application
+--			Consent device-> Activate application
+--
+--     2. Performed steps
+--			Perform PTU that contains policies for app with <functional grouping> that lacks "user_consent_prompt" field		   
+--
+-- Expected result:
+--		PoliciesManager must apply such <functional grouping> without asking User`s consent for it	
+---------------------------------------------------------------------------------------------
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: Please clarify the <such> part with 

>  <functional grouping> that lacks "user_consent_prompt"

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+		end)
+		self.mobileSession:ExpectResponse(correlationId, { success = true, resultCode = "SUCCESS" })
+		self.mobileSession:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+	end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:Step1_PTU_lack_of_user_consent_prompt()
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: Please rename to 
function Test:TestStep_PTU_lack_of_user_consent_prompt()

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+			end			
+			RUN_AFTER(to_run, 500)
+		end)
+		EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", {appID = self.HMIAppID,  appPermissionsConsentNeeded = false})
+		self.mobileSession:ExpectResponse(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+		end)
+	end)
+end
+
+function Test:Step2_Activate_app_no_consent()
```
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: Please see comment above. All tests in this section should follow the convention prefix TestStep_ + description

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua [hunk 5]
```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------
```
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: accepted

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 1 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: rejected, on PTU step also is checked that param "appPermissionsConsentNeeded" is "false"

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 2 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: rejected, lacks "user_consent_prompt" in mentioned in description

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 3 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: accepted

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 4 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: accepted

**Comment by aarotar** [comment]
@maria-mih, updated in commit  7e06ebc

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 1 above)
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: I meant that in preconditions you need to add preparation of such PT.

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 5 above)
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: Please delete the empty spaces in section Description. 
All lines should be left aligned with no spaces as shown in template https://github.com/CustomSDL/sdl_atf_test_scripts/pull/32

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 2 above)
Comment (maria-mih) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: Ok if you believe that such is enough as description and it is too difficult to make it 
PoliciesManager must apply <functional grouping> lacking  "user_consent_prompt" without asking User`s consent for it

Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: rejected. Actually, it is interesting, what do you understand under "PoliciesManager must apply lacking "user_consent_prompt"... and if any other test engineer will understand what is it ))))

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 1 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: rejected.

File: test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua (hunk 5 above)
Comment (aarotar) [inline test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua]: rejected.

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**